cmd/registry/cmd/root.go

```diff
@@ -32,10 +32,13 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var Version = "dev"
```

Comment:
Is this being overridden somewhere in the build process?

Reply:
Here? 
```
ldflags:
  - -s -w -X "github.com/apigee/registry/cmd/registry/cmd.Version={{.Env.RELEASE_VERSION}}"
```

Reply:
Yes. we are pulling the RELEASE_VERSION from the tag name in the release.yml workflow file

Reply:
I think we should add a comment to explain the purpose here.

Reply:
Agreed, when I first saw this in the code I thought it was unfinished.

Reply:
comment added